### DISCUSSION TITLE
Where to hold General Discussions about yesvelte

### DISCUSSION THREAD

**Discussion by leejoramo-d51** [yesvelte/yesvelte#239] [General]
I am reviewing `yesvelte`  to see how it meets my needs. Very impressive. 

I have previously used `mdbsvelte` and `sveltestrap`.  And I have reviewed most of  the other Svelte CSS/Forms tools. I like your design choices to create a flexible and extendible system. _It's \<el> tags all the way down!_ Access to Tabler's extensive collection of components is interesting.

I am working a schema-based form system for Svelte that supports reactivity, form validation and more. I am already testing migrating to yesvelte's components. (For a description of  scheme-based forms: https://www.taniarascia.com/schema-based-form-system/ )

I have a bunch of general questions about future directions, how-to implement my own components.

But first where do you really want such a discussion? Since you have activated GitHub Discussions, this would be my natural first choice. But there is no current discussions here. I see you point to Reddit and many projects seem to focus on GitHub Issues. 

Thanks

**Comment by pournasserian** [comment]
> I am reviewing `yesvelte` to see how it meets my needs. Very impressive.
> 
> I have previously used `mdbsvelte` and `sveltestrap`. And I have reviewed most of the other Svelte CSS/Forms tools. I like your design choices to create a flexible and extendible system. _It's <el> tags all the way down!_ Access to Tabler's extensive collection of components is interesting.
> 
> I am working a schema-based form system for Svelte that supports reactivity, form validation and more. I am already testing migrating to yesvelte's components. (For a description of scheme-based forms: https://www.taniarascia.com/schema-based-form-system/ )
> 
> I have a bunch of general questions about future directions, how-to implement my own components.
> 
> But first where do you really want such a discussion? Since you have activated GitHub Discussions, this would be my natural first choice. But there is no current discussions here. I see you point to Reddit and many projects seem to focus on GitHub Issues.
> 
> Thanks

Thanks for the feedback! 

We usually use Reddit to share the ideas and suggestions. [Here is the YeSvelte Reddit](https://www.reddit.com/r/yesvelte/) which we have created recently.

Also, our team can continue discussion here as well.
